elizhahn/niche-audience
# Feature/total pack weight

## What does this PR do (summary of changes)?
- A PackStatistics component was added to render the weight data for the user
- As a user enter's their shelf items, they will now see a pack total in Oz and Lbs update
- The weight will dynamically render when a user adds an item or removes an item
- User's are prevented from entering a negative amount
- Initial styling was added to the statistics area
## How should it be tested?
- Open up a few shelves and add items, you should see the weight add up on the right side
- Additionally, remove the items, you should see the weight decrease
- When all items are removed, the weight should render 0 Oz and 0Lbs
## Future Iterations:
- Styling needs to be tweeked
